includes/constants/class-country-test-cards.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Class Country_Test_Cards
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Constants;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+/**
+ * Class handling country-specific data for WooPayments
```

Comment:
```suggestion
 * Class handling country-specific test card numbers for WooPayments
```

I am thinking maybe the description is too generic? 🤷